mathlibtools/file_status.py

```diff
@@ -50,6 +50,9 @@ def pr_link(self) -> Optional[str]:
             return None
         return f"https://github.com/leanprover-community/mathlib4/pull/{self.mathlib4_pr}"
 
+    def to_gexf(self) -> str:
+        return repr(self)
```

Comment:
What's your plan for parsing this in the consumer? It might make more sense to unpack it into a series of separate attributes, or maybe serializing to json in some way.

Reply:
I don't know what downstream gexf uses are -- Patrick brought up to me that gexf export was completely blocked by file statuses. The export needs a very basic type, str or numeric, doesn't even support lists. So I think one solution would have been to just strip statuses completely in the gexf export.

If an end user has a gexf application/use, I hope they file an issue on making the output easier to parse.

Reply:
Ah that makes sense, thanks!